visualization/app/codeCharta/state/store/files/files.splitter.ts

```diff
@@ -0,0 +1,7 @@
+import { setFiles } from "./files.actions"
+import { CCAction } from "../../../codeCharta.model"
+import { Files } from "../../../model/files"
+
+export function splitFilesAction(payload: Files): CCAction[] {
+	return [setFiles(payload)]
+}
```

Comment:
Why do we need to return an array of CCActions..? Might be correct, but I need some clarification :)

Reply:
Splitter is required when we want to update the whole state object and split each property to delegate it to their reducer. We implemented it the way, that splitting an action returns an array of CCActions, even though there is only one action when there are no more nested reducers (see blacklist or files).

Reply:
oh yea right, because its on the same level as the `dynamicSettings.splitter` etc. Thanks! (: